controllers/user/controllers/account_controller.go

```diff
@@ -93,23 +89,10 @@ func (r *AccountReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ct
 	account := &userv1.Account{}
```

Comment:
Does the latter feature want to be completely removed? Now it seems that there is still part of it in the user module.

Reply:
the next step will completely removed account from user.

Reply:
next pr will do this and add e2e test